### DISCUSSION TITLE
鲨鲨和他的粉丝交流群

### DISCUSSION THREAD

**Discussion by Sharkshark-zzz** [share121/inter-knot#4644] [绝区零]
![mmqrcode1725811576569](https://github.com/user-attachments/assets/ffaebb38-352c-4302-a74a-e6e0953b77d1)
欢迎各位绳匠来鲨鲨的秘密基地一起happy～和大家友好相处

**Comment by Sharkshark-zzz** [comment]
顶一下～

**Comment by Sharkshark-zzz** [comment]
b站搜鲨鲨-绝区零

**Reply by A-Rys** [reply]
抓~
